
recording : True

recorder_folder : "/home/ros/ros_ws/src/i_am_project/data/airhockey/"
